src/plugins/workspace/public/components/workspace_form/workspace_form.tsx

```diff
@@ -48,7 +48,7 @@ export const WorkspaceForm = (props: WorkspaceFormProps) => {
   const disabledUserOrGroupInputIdsRef = useRef(
     defaultValues?.permissionSettings?.map((item) => item.id) ?? []
   );
-  const isDashboardAdmin = application?.capabilities?.dashboards?.isDashboardAdmin ?? false;
+  const isDashboardAdmin = !!application?.capabilities?.dashboards?.isDashboardAdmin;
```

Comment:
Why we need to change this?

Reply:
it will throw an error ```
Type 'true | Readonly<{ [x: string]: boolean; }>' is not assignable to type 'boolean'.
  Type 'Readonly<{ [x: string]: boolean; }>' is not assignable to type 'boolean'.
``` if we don't change this

Reply:
We don't need to pass it to other component anymore. So I think we can remove this change.